### PR TITLE
Tooltips: improve wording when move is weakened by ability

### PR DESCRIPTION
**Pull request by asgdf**
Mostly applies to Rivalry.

### CODE REVIEW

**Comment by Zarel** [comment]
I like this pullreq more (in terms of code readability), but I like my suggestion in #926 even more.

**Comment by Zarel** [comment]
"×0.75 from Rivalry" is I think better phrasing.

**Comment by Zarel** [comment]
(Discuss.)

**Comment by urkerab** [comment]
Other than "(Technician boosted)", everything else already uses "(Boosted by ...)" or "(Reduced by ...)", so that wording would at least be consistent.

**Comment by asgdf** [comment]
In particular, there's "Boosted by status" and "Boosted by lack of item" that don't quite work with "x2 from [...]", so either we'll have to use something else or be somewhat inconsistent.

**Comment by Zarel** [comment]
"×2 from lack of item"

"×1.5 from status"

Seems fine to me?

**Comment by asgdf** [comment]
But like, does the boost come *from* status, or does it rather come from Venoshock because of the status, for example? Maybe I'm overthinking this.

**Comment by Zarel** [comment]
you can do "from activated Venoshock" and "from activated Unburden"

**Comment by asgdf** [comment]
What about Wake-Up Slap and Smelling Salts? "×2 from activated Smelling Salts"? I think it's a bit unfortunate that the tooltip doesn't actually say what exactly *causes* the boost anymore then.
Would it be okay to have both "×2 from [ability]" and "×2 due to [condition]"?

**Comment by Zarel** [comment]
"×2 from Smelling Salts + slp" or whatever is fine, but I think explaining how moves/abilities activate is maybe beyond the scope of a move tooltip...

**Comment by asgdf** [comment]
Okay. Hope it's a good balance of conciseness and looking sufficiently pretty for end-user output now.

File: js/client-battle-tooltips.js [hunk 1]
```diff
@@ -883,6 +883,10 @@ var BattleTooltips = (function () {
 		return moveType;
 	};
 
+	BattleTooltips.prototype.makePercentageChangeText = function (boost, source) {
+		return ' (×' + boost + ' from ' + source + ')';
```
Comment (Zarel) [inline js/client-battle-tooltips.js]: Unicode in client code reeeeeee

Comment (Zarel) [inline js/client-battle-tooltips.js]: Seriously, if this field accepts HTML, do `&times;`

Comment (Zarel) [inline js/client-battle-tooltips.js]: Otherwise do `\u...`

**Comment by asgdf** [comment]
Woops. Fixed.

**Comment by Zarel** [comment]
I don't know if I _like_ this solution but I don't dislike it.

Good enough, I guess.

**Pull request merged**